### ISSUE TITLE
[Request] - Train a model for upscaling movie images / movies

### ISSUE THREAD

**Issue by Felixkruemel** [jixiaozhong/RealSR#7]
I would like to see a model which has been trained to upscale movies.
Currently DF2K-JPEG works really good on pictures with compression artifacts (it's like magic, wow). But for real life footage which is older (like on DVDs or old movies) it isn't really that big of a difference compared to non-upscaled footage.

So is there a possibility we could get a movie model in the future?

**Comment by jixiaozhong** [comment]
Yes, we will improve the speed and stability when testing on videos.

**Comment by Felixkruemel** [comment]
@jixiaozhong 
Are there any news regarding this?

**Comment by ElaineGxy** [comment]
Hi, I know you run DF2K-JPEG successfully, but when I used DF2K-JPEG model ,I got an OOM error like this:


![test_error](https://user-images.githubusercontent.com/10204869/87402845-b3f09c00-c5ee-11ea-875d-2a3a5df4349e.png)

can you give me some advice about this ,thank you!

**Comment by Felixkruemel** [comment]
@ElaineGxy That has nothing to do with my issue.

Still your problem is just that your GPU doesn't have enough VRAM. Try a smaller tile size.

**Comment by Felixkruemel** [comment]
@jixiaozhong Any news?
